test/issues/general/test_5703.test

```diff
@@ -0,0 +1,63 @@
+# name: test/issues/general/test_5703.test
+# description: Issue 5703: Can't have a window aggregate and unnest.
+# group: [general]
+
+statement error
+select unnest([5,4,3]) as r, sum(r) OVER ();
+----
+Binder Error
+
+statement error
+select unnest([5,4,3]) as r, sum(r) OVER () + 42;
```

Comment:
What happens if you have the following query:

```sql
select unnest([5,4,3]) as r, sum(r + 42) OVER ();
```

Reply:
This also fails. Have added a test